### PR TITLE
3tube sat

### PR DESCRIPTION
**Pull request by smsimon**

### CODE REVIEW

**Comment by smsimon** [comment]
@keskitalo not sure why one of the tests failed as I only changed a couple of numbers in the config.py file

**Comment by tskisner** [comment]
That looks like a transient network connection failure.  I'll try to restart it.

File: s4sim/hardware/config.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2020 CMB-S4 Collaboration.
+# Copyright (c)
```
Comment (keskitalo) [inline s4sim/hardware/config.py]: Any particular reason for trimming the copyright statement?

Comment (smsimon) [reply s4sim/hardware/config.py]: just an old copy before we had those likely

**Comment by keskitalo** [comment]
I fixed the issue with the wafer selection problem. I'm going to ask Ted to take a look at the new `Hardware.select()` method.

File: s4sim/hardware/config.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def select(self, telescopes=None, tubes=None, match=dict()):
             if wselect is None:
                 # Just the regular behavior
                 if isinstance(v, list):
-                    reg[k] = re.compile(r"(" + "|".join(v) + r")")
+                    reg[k] = re.compile(r"(^" + "$|^".join(v) + r"$)")
```
Comment (tskisner) [inline s4sim/hardware/config.py]: Thanks for fixing the regex matching here!

**Comment by keskitalo** [comment]
Merging after two reviews.

**Pull request merged**